### DISCUSSION TITLE
Arduino: soc/i2s_periph.h Error When Building Any Example

### DISCUSSION THREAD

**Discussion by whiteshavedchocolate** [mrcodetastic/ESP32-HUB75-MatrixPanel-DMA#581] [Q&A]
I've been using the Arduino SmartMatrix library for a few years, successfully driving different LED modules. I wanted to try out the ESP32-HUB75-MatrixPanel-DMA library, as it has support for the ESP32-S3.

I feel like a moron, but after installing this library in Arduino on my Windows 11 laptop, I cannot successfully build any of the examples. I get this error:
...ESP32_HUB75_LED_MATRIX_PANEL_DMA_Display\src/platforms/esp32/esp32_i2s_parallel_dma.hpp:56:54: fatal error: soc/i2s_periph.h: No such file or directory

I did a search in the "src" folder, and I'm unable to find i2s_periph.h. I'm guessing I'm missing something really basic, but I'm stumped. Anybody know what's up? Thank you.

**Comment by Bsm-B** [comment]
just update the ESP32 board library  version
